php_gearman.c

```diff
@@ -3613,10 +3613,6 @@ static void gearman_client_obj_free(zend_object *object) {
 	zval_dtor(&intern->task_list);
 
 	zend_object_std_dtor(&intern->std);
-
-	// I'm including it because it cleans up a memory leak, but still not sure why it's not automatically
-	// cleaned up like other objects. Need to dig in more
-	efree(intern);
```

Comment:
I don't think we can leave a memory leak in. I agree that I'm not 100% a fan of the efree, but it needs to be better addressed than this